shared/teams/team/container.js

```diff
@@ -105,10 +78,5 @@ export default compose(
       }
     },
   }),
-  // TODO remove these branches, let's not send all these props to all possible outcomes
```

Comment:
the branches and hocs in general make this harder to understand and harder to flowtype